pkg/request/request.go

```diff
@@ -14,7 +14,7 @@ func DeleteRequest(url string, dryRun bool, connection *sdk.Connection) error {
 		return fmt.Errorf("can't parse url '%s': %v\n", url, err)
 	}
 	if dryRun {
-		fmt.Printf("DRYRUN: Would have called %v.\n", request.GetPath())
+		fmt.Printf("DRYRUN: Would have called DELETE on %v.\n", request.GetPath())
```

Comment:
The dot at the end of the line make it harder to read or copy/paste if needed. I would remove it.

Reply:
removed